contracts/mocks/EnumerableSetMock.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.5.10;
+import "../utils/EnumerableSet.sol";
+
+
+/**
+ * @title EnumerableSetMock
+ * @dev Data structure - https://en.wikipedia.org/wiki/Set_(abstract_data_type)
+ * @author Alberto Cuesta Cañada
+ */
+contract EnumerableSetMock{
+
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet private set;
+
+    constructor() public {
+        set = EnumerableSet.AddressSet({values: new address[](0)});
```

Comment:
Can we add a `new` function, to make usage easier? It'd have to be specialized for the type however, so `EnumerableSet.newAddressSet` maybe?

Reply:
Done.